File: exo_09.exs
year = IO.gets("En quelle année es-tu né(e) ?\n> ") |> String.trim("\n")
i = String.to_integer(year)
date = DateTime.utc_now() |> Map.fetch!(:year)

cond do
  String.length(year) == 0 ->
    IO.puts("Et ton année de naissance ?\n")

  true ->
    for i <- i..date, i <= date, do: IO.puts(i)
end

File: exo_09.rb
print "En quelle année es-tu né(e) ?\n> "
year = gets.chomp.to_i

if year.zero?
  abort("Et ton année de naissance ?\n")

else if year == Time.new.year
  abort("Tu es trop jeune !\n")

end
end

while year <= Time.new.year
  puts year
  year += 1
end
